packages/tailwind/src/kz-spacing.ts

```diff
@@ -1,14 +1,41 @@
 import { heartTheme } from "@kaizen/design-tokens"
 
+const tokenDenyList: string[] = [
+  "xs",
+  "sm",
+  "md",
+  "lg",
+  "xl",
+  "xxl",
+  "xxxl",
+  "xxxxl",
+  "xxxxxl",
+  "xs-id",
+  "sm-id",
+  "md-id",
+  "lg-id",
+  "xl-id",
+  "xxl-id",
+  "xxxl-id",
+  "xxxxl-id",
+  "xxxxxl-id",
+]
+
+/**
+ * checks if a key from design tokens should be filtered from tailwind
+ */
+const isValidToken = (key: string | number): boolean => {
+  if (tokenDenyList.indexOf(`${key}`) < 0) return true
```

Comment:
```suggestion
  return tokenDenyList.indexOf(`${key}`) >= 0
```
would work as well ;)

Reply:
oooo yes, that's much better 😉

Reply:
Actually, scratch that, it should be
```
return !tokenDenyList.includes(`${key}`)
``

Reply:
Updated 👍 though I've returned ```tokenDenyList.indexOf(`${key}`) < 0``` instead of a >= since this checking to see if the key does not exist in the tokenDenyList so the index should be -1 for valid tokens and return true

Reply:
sorry just refresh the page and realised there was an additional comment 😆 I've always leaned on indexOf because it meant to be slightly more performant. Not wedded to it so I can change it to include for readabilities sake